packaging/torcharrow/meta.yaml

```diff
@@ -11,49 +11,39 @@ requirements:
     # - { { compiler('cxx') } } # [win]
     - cmake
     - python
-    {% if osx %}
-    - boost
-    {% elif linux %}
-    - ninja-build
-    - checkinstall
-    - libssl-dev
-    - libboost-all-dev
-    - libdouble-conversion-dev
-    - libgoogle-glog-dev
-    - libgflags-dev
-    - libevent-dev
-    - libre2-dev
-    {% endif %}
   host:
     - python
     - setuptools
     {{ environ.get('CONDA_PYTORCH_BUILD_CONSTRAINT') }}
   run:
     - python
     - numpy
-    - boost
-    - libboost
     - typing_inspect
     - pandas
     - typing
     - tabulate
     - pyarrow
     - arrow
     - cffi
+    - glog 0.4.0
     {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
 
 build:
   string: py{{py}}
   script_env:
     - BUILD_VERSION
 
+#GitHub Actions usually provide Mac without AVX2 support, for now just skip the test until Velox can be run without AVX2
+#pytest --no-header -v torcharrow/test
```

Comment:
I think Velox team has solved this issue.

Reply:
nope the issue was on some machines with AVX2 support velox still encounters issues sometimes, but if the machine doesn't support AVX2 in the first place (which is the case for Github Action mac machines), it's guaranteed to fail.

Reply:
you mean the ones with AVX2 support but still fails? I haven't met those on Github Actions (since they don't support AVX2 anyways). Here's an early test that I did have the "illegal instruction" errors, it happened when doing the import torcharrow after building: https://github.com/facebookresearch/torcharrow/runs/6297806435?check_suite_focus=true . Please note that the workflow file is different from the one in this PR, but the main structure for macos-container should be almost the same.

Reply:
Yeah, https://github.com/facebookresearch/torcharrow/pull/315 and https://github.com/facebookincubator/velox/pull/1549 solves the issue that Velox requires AVX512. But still needs AVX2 :)

Reply:
We can actually support cpus without avx now (thats how we will support m1 etc). Our build scripts havent caught up with it, but I will make those changes shortly.

Reply:
nice, I'll enable the mac tests when it's ready.